CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 2.8.11)
 
+if(BIICODE)
+   include("biicode.cmake")
```

Comment:
So I'm still learning how Biicode works so please bear with me. I'm reading through

http://docs.biicode.com/c++/adapt_library_guide.html#option-1-let-biicode-do-its-job-in-an-isolated-file

It looks like we are using option 1 here (and I like it better), but option 1 doesn't seem to need a [dependencies] section in biicode.conf (option 2 does though). Can we remove the dependencies secton?

Also let's use 2-space indentation for consistency (also in the conf file). I've fixed the indentation at line 9 of this file. Thanks!